docs/DeveloperGuide.adoc

```diff
@@ -75,21 +83,27 @@ The sections below give more details of each component.
 === UI component
 
 .Structure of the UI Component
+
+[NOTE]
+UI image and description to be updated with the later milestones.
+
 image::UiClassDiagram.png[]
 
 *API* : link:{repoURL}/src/main/java/seedu/address/ui/Ui.java[`Ui.java`]
 
-The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class.
+The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `EntityListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class.
 
 The `UI` component uses JavaFx UI framework. The layout of these UI parts are defined in matching `.fxml` files that are in the `src/main/resources/view` folder. For example, the layout of the link:{repoURL}/src/main/java/seedu/address/ui/MainWindow.java[`MainWindow`] is specified in link:{repoURL}/src/main/resources/view/MainWindow.fxml[`MainWindow.fxml`]
 
 The `UI` component,
 
 * Executes user commands using the `Logic` component.
-* Listens for changes to `Model` data so that the UI can be updated with the modified data.
+* Looks at the prefix commandType given in the CommandResult(returned after every Command is executed), as displays the specific EntityListPanel respectively.
 
 [[Design-Logic]]
 === Logic component
+[NOTE]
+Logic Architecture image and description to be updated with the later milestones.
 
 [[fig-LogicClassDiagram]]
 .Structure of the Logic Component
```

Comment:
The Architecture Diagram here contains words that are too small to see. Try breaking it up into separate diagrams to improve readability.